src/firefly/js/ui/DownloadDialog.jsx

```diff
@@ -262,20 +262,17 @@ export function TitleField({style={}, value, label='Title:', size=30}) {
     );
 }
 
-export function ZipStructure({fieldKey='zipType'}) {
+export function FileStructure({fieldKey='zipType'}) {
     return (
-        <ListBoxInputField
-            fieldKey = {fieldKey}
-            initialState = {{
-                tooltip: 'File Structure',
-                label : 'File Structure:'
-            }}
-            options = {[
-                {label: 'Structured (with folders)', value: 'folder'},
-                {label: 'Flattened (no folders)', value: 'flat'}
-            ]}
-        />
-
+        <Stack width={250} mt={2}>
```

Comment:
```suggestion
        <Stack>
```
[Nitpick] don't constraint the width unless it's overflowing and you don't need margin top, the parent stack has spacing={2} set so that will take care of this.

I wonder if we really need to wrap single child (Switch component) in Stack. I'd just take it out.